operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -1012,8 +1012,8 @@ private static Step bringAdminServerUp(DomainPresenceInfo info, Step next) {
 
   private static Step[] bringAdminServerUpSteps(DomainPresenceInfo info, Step next) {
     List<Step> resources = new ArrayList<>();
-    resources.add(PodHelper.createAdminPodStep(null));
     resources.add(new BeforeAdminServiceStep(null));
```

Comment:
Should we rename this to be BeforeAdminPodStep instead of BeforeAdminServiceStep?

Reply:
@rjeberhard is this something you think needs renaming

Reply:
I kind of remember discussing this and suggesting BeforeAdminStep.